site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -305,17 +305,17 @@ The following tables lists the configurable parameters of the Agones chart and t
-| `agones.extensions.validatingWebhook.annotations`        | [Annotations][annotations] added to the Agones validating webhook                                                                                                                                                                                                                                                | `{}`                                      |
-| `agones.extensions.validatingWebhook.disableCaBundle`    | Disable ca-bundle so it can be injected by cert-manager                                                                                                                                                                                                                                                          | `false`                                   |
-| `agones.extensions.mutatingWebhook.annotations`          | [Annotations][annotations] added to the Agones mutating webhook                                                                                                                                                                                                                                                  | `{}`                                      |
-| `agones.extensions.mutatingWebhook.disableCaBundle`      | Disable ca-bundle so it can be injected by cert-manager                                                                                                                                                                                                                                                          | `false`                                   |
+| `agones.extensions.allocationApiService.annotations`     | [Annotations][annotations] added to the Agones API registration. If not set, this parameter defaults to the value of `.Values.agones.controller.allocationApiService.annotations`.                                                                                                                                                                                                                                                   | `{}`                                      |
+| `agones.extensions.allocationApiService.disableCaBundle` | Disable ca-bundle so it can be injected by cert-manager. If not set, this parameter defaults to the value of `.Values.agones.controller.allocationApiService.disableCaBundle`.                                                                                                                                                                                                                                                          | `false`                                   |
```

Comment:
```suggestion
| `agones.extensions.allocationApiService.disableCaBundle` | Disable ca-bundle so it can be injected by cert-manager.| `false`                                   |
```

Good idea on the explanation of what happens, but rather than explain it here, let's go back to each matching `agones.controller....` property above, and prefix it with "Deprecated. Please use `agones.extenions....` instead".

So for this one, we would have `agones.controller.allocationApiService.disableCaBundle` be:

```
| `agones.controller.allocationApiService.disableCaBundle` | **Deprecated**. Use `agones.extensions.allocationApiService.disableCaBundle` instead. Disable ca-bundle so it can be injected by cert-manager                                                                                                                                                                                                                                                          | `false`                                   |
```

Does that make sense?

Reply:
This is much clearer, and I've updated everything accordingly.